.github/CODEOWNERS

```diff
@@ -137,9 +137,9 @@ CHANGELOG*
 /x-pack/filebeat/input/salesforce @elastic/obs-infraobs-integrations
 /x-pack/filebeat/input/streaming/ @elastic/security-service-integrations
 /x-pack/filebeat/module/activemq @elastic/obs-infraobs-integrations
-/x-pack/filebeat/module/aws @elastic/obs-cloud-monitoring
-/x-pack/filebeat/module/awsfargate @elastic/obs-cloud-monitoring
-/x-pack/filebeat/module/azure @elastic/obs-cloud-monitoring
+/x-pack/filebeat/module/aws @elastic/obs-ds-hosted-services
+/x-pack/filebeat/module/awsfargate @elastic/obs-ds-hosted-services
```

Comment:
IIRC, the `awsfargate` ownership is now on the integrations team.

Reply:
Does your team own both integration and metricset or do we need to keep this split?

Reply:
@lalit-satapathy, I'll go ahead with this update. 

Let me know your thoughts about the `awsfargate` metricset when you have time.